doc/examples/features_detection/plot_censure.py

```diff
@@ -31,12 +31,18 @@
 ax[0].axis('off')
 ax[0].scatter(detector.keypoints[:, 1], detector.keypoints[:, 0],
               2 ** detector.scales, facecolors='none', edgecolors='r')
+ax[0].set_title("Original Image")
 
 detector.detect(img_warp)
 
 ax[1].imshow(img_warp, cmap=plt.cm.gray)
 ax[1].axis('off')
 ax[1].scatter(detector.keypoints[:, 1], detector.keypoints[:, 0],
               2 ** detector.scales, facecolors='none', edgecolors='r')
+ax[1].set_title("Transformed Image")
+
+ax[1].text(372, 450, 'AffineTransform scale = 1.5', ha='center', va='center', color='yellow')
+ax[1].text(325, 475, 'AffineTransform rotation 28.6 degrees', ha='center', va='center', color='yellow')
+ax[1].text(290, 500, 'AffineTransform translation X = 150  Y = -200', ha='center', va='center', color='yellow')
```

Comment:
I think it'd be better to output this text via `Output` cells (i.e. just use `print` function).
See the last plot and the cell below it in http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_local_binary_pattern.html .